### DISCUSSION TITLE
Incorrect amount paid as reward of dynamic staking

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#99] [Bugs and Reports]
pls check this wallet
0xAE5A96A197F7F7bAE36F5E32BE24532a61Bc6115

he did not recirve 3.65% this dynamic cycle but only 2.86%

![IMG_1021](https://github.com/user-attachments/assets/ec25b9ff-5dd0-44d9-b8e6-cd81cd9c43e0)

**Comment by bf-mike-mkt** [comment]
do we have an update pls?

**Comment by bf-monster** [comment]
closing as stale
